components/button/button-subtle.js

```diff
@@ -1,7 +1,9 @@
 import { ifDefined } from 'lit-html/directives/if-defined.js';
 import { LitElement, html } from 'lit-element/lit-element.js';
-import { D2LButtonMixin } from './button-mixin.js';
-import { buttonStyles } from './button-styles.js';
+import { label } from '../typography/styles.js';
```

Comment:
I am not completely in love with the nomenclature here.  Another option is to move the label and shared stylesheet import into the `button-subtle-styles.js` and change it to export the compete array.  The thing I don't like about that however, is that we end up with some style modules exporting a CSS template result, and others exporting an array of CSS template results.

Reply:
Like for `label` specifically it should probably end up in the future fields/fieldsets/labels set of components.

Reply:
Yeah agreed - I was just leaving them with typography for the moment since that's where it came from.  What do you think of the imported names though?  The names (`buttonSubtle`, `button`, `label`) don't really make it clear what these imported things are...

Reply:
Hmm, I don't mind it. `style.js` on its own doesn't work right? So maybe that could be `button-shared-styles.js`? And then yeah maybe I'll go back on my previous comment and agree that having `-styles.js` at the end is handy, so label would become `import {labelStyles } from '../typography/styles.js';`. That would in theory let you mix and match and have style exports and normal exports in the same file.

Reply:
Yeah I think so.  I'll switch it up and see how it goes.

Reply:
Yeah, I think those updates sound better.  I agree with Dave B, I don't really love the way this looks, but it also may just take some getting used to.  If we had 20 components all following the same convention, I think it'd feel less weird